app/views/breeders/show.html.erb

```diff
@@ -12,7 +12,7 @@
 
   <h3>Litters</h3>
   <% @breeder.litters.reverse_each do |litter|%>
-    <h2><%= link_to litter.name, breeder_litters_path(@breeder.id, litter.id) %></h2>
+    <h2><%= link_to litter.name, litter_path %></h2>
```

Comment:
I tried on my local machine, I think we need to pass the litter.id into the path, otherwise, it will always navigate the same litter <%= link_to litter.name, litter_path(litter.id) %>

Reply:
@xinglin2016 Awesome thanks for point that out man. I just pushed it again with the fix you suggested. Tested locally and it looks good now.

Reply:
Great